### ISSUE TITLE
questions on fine tuning,word embedding,

### ISSUE THREAD

**Issue by prakashwarbler** [flairNLP/flair#1473]
1. Trained the model with custom dataset in LSTM with CRF.  glove embedding is used but accuracy is poor. do i need to create my own word embedding for cust dataset?  does it improve the accuracy?
2. I have very limited sentences in dataset. so i stacked with conell 2003 dataset to improve the accuracy. I have keywords of 11000 plus for each entity like PER,LOC,ORG. some of the entity keywords are more than 4 to 5 words. example : "Ragaventhra  subramaniya iyyer pillai"- PER creating synthetic dataset is the only option for these kind of scenario? need your best suggestion?
3. Changing dropout value will improve accuracy? by default, it is 0.5. ?
4.  70% training dataset, 20% testing and 10% for validation. this is the distribution of data. can i play around here by changing the percentage? will it help to improve the accuracy?
5. is there a specific way to improve the accuracy by providing synthetic dataset? 
6. shall I stop the training if it shows keep on BAD EPOCH = 1,2,3,4 for more than 2 to 3 times?
7.  How about case sensitivity? do i need to add both upper and lower case of same word to get result on upper and lower case word? or any one is more than enough?
8. By default it is 150 EPOCH. but last training has been stopped automatically at 99 EPOCH with error of  "learning rate too small - quitting training!" . how to rectify this error?
9.  what is the best optimiser you suggest for flair -NER  LSTM-CRF model?

**Comment by prakashwarbler** [comment]
anyone here to ans?
